src/index.d.ts

```diff
@@ -699,6 +699,12 @@ declare namespace RamdaAdjunct {
             (value: any): (obj: object) => boolean;
         };
 
+        /** 
+         * Checks if input value falls within the supplied range. The bottom of the range is
+         * inclusive, the top of the range is exclusive.
+         */
+        inRange(low: number, high: number, value: number): boolean;
```

Comment:
You have to create curried alternatives.

This is how to do that when you have function with 3 arguments: https://github.com/char0n/ramda-adjunct/blob/b9ac42e0a182958f5930218b4a3bde900c7be4ea/src/index.d.ts#L684